src/mrinufft/trajectories/trajectory3D.py

```diff
@@ -149,6 +150,180 @@ def initialize_3D_cones(
     return trajectory.reshape((Nc, Ns, 3))
 
 
+def initialize_3D_helical_shells(Nc, Ns, nb_shells, oversampling=1, tilt="uniform"):
+    """Initialize 3D trajectories with helical shells.
+
+    Parameters
+    ----------
+    Nc : int
+        Number of shots
+    Ns : int
+        Number of samples per shot
+    nb_shells : int
+        Number of concentric shells/spheres
+    oversampling : int
+        Multiplicative factor for Ns without changing the curves, by default 1
+    tilt : str, float, optional
+        Angle between shots over a shell surface, by default "uniform"
+
+    Returns
+    -------
+    array_like
+        3D helical shell trajectory
+    """
```

Comment:
Don't forget to add a Reference section, with citations. A Notes section with the history mentioned in the PR can also be added.

Reply:
Done